Lib/test/test_posixpath.py

```diff
@@ -749,6 +751,9 @@ def check_error(exc, paths):
         self.assertRaises(TypeError, posixpath.commonpath,
                           ['usr/lib/', b'/usr/lib/python3'])
 
+        # gh-117201: Handle leading `//` for `posixpath.commonpath`
+        check(['//foo/bar', '//foo/baz'], '//foo')
```

Comment:
Are there tests for mixed `//` and `/` rooted paths?

Please add also tests for `///` prefix.

Reply:
There was one, but I have now grouped all tests for the leading slashes together.